src/text_extraction/text_extractor.py

```diff
@@ -12,22 +13,47 @@ class TextExtractor():
     def __init__(self):
         self.out_dir = ""
         self.dpi = 500
+        self.metadata_handler = MetadataHandler()
 
-    def read(self, input_file):
+    def read(self, input_file, file_name=None, index=None, uploader=None):
```

Comment:
Hvorfor kan input_file ikke bruges i stedet for "file_name"?

Reply:
Input_file er til at vise stien til input image file, brugt af OCR og bliver også brugt til at åbne billedet.

file_name er sat til at give navnet på selve filen.

Reply:
Og du kan ikke bare gøre som der bliver gjort på linje 20?

Reply:
Du har dælme ret, os.path.basename(input_file) Finder jo basename selvfølgelig.